DemoServer/Controllers/Demos/TextSearch/HighlightResultsMapIndex/metadata.json

```diff
@@ -0,0 +1,286 @@
+                "",
+                "  * `TermVector()`<br>",
+                "     Set _FieldTermVector.WithPositionsAndOffsets_ to store the term's position and offset from the text.",
+                "",
+                "* During the indexing process, the fields' content will be tokenized, broken into distinct terms according to the configured analyzer (RavenStandardAnalyzer is used by default).",
+                "",
+                "* A query can now be made on these terms to get their matching documents including a list of text fragments containing the highlighted searched terms. See the following steps."
+            ],
+            "Assets": [
+                {
+                    "Title": "FTS - Static Index Demo",
+                    "Type" : "demo",
+                    "Url"  : "/text-search/fts-with-static-index-single-field"
+                },
+                {
+                    "Title": "Store Fields on Index Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/store-fields-on-index"
+                },
+                {
+                    "Title": "Querying highlighting",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/Csharp/indexes/querying/highlighting"
+                },
+                {
+                    "Title": "Indexes - Analyzers",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/using-analyzers"
+                },
+                {
+                    "Title": "Using term vectors",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/Csharp/indexes/using-term-vectors"
+                },
+                {
+                    "Title": "Term vector in Lucene",
+                    "Type" : "link",
+                    "Url"  : "http://makble.com/what-is-term-vector-in-lucene"
+                }
+            ]
+        },
+        {
+            "Title": "Define highlight tags",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Prior to making the search query, you can define the tags that will mark the searched terms within the text fragments. This is an optional step.",
+                " ",
+                "* _To define your own tags_ create a `HighlightingOptions` object with _PreTags_ & _PostTags_.<br>",
+                "  This object is passed to the query - see next step.",
+                "",
+                "* _The default pre & post tags_ are: `<b style='background:yellow'>......</b>`<br>",
+                "  The color will change per field that is searched.",
+                "",
+                "* When defining multiple tag strings in the tags array,<br> and when querying with multiple highlight field requests,<br> then each tag string is used per field requested respectively."
```

Comment:
I cannot comprehend the meaning of this. Could you clarify it somehow, please?

Reply:
as discussed: removed this bullet + separated demos